pytket/docs/changelog.rst

```diff
@@ -9,10 +9,12 @@ Features:
 * Add ``OpType.CnRx`` and ``OpType.CnRz``.
 * Add ``AutoRebase`` and ``AutoSquash`` passes.
   Deprecate ``auto_rebase_pass`` and ``auto_squash_pass``.
+* Add new parameter added to `remove_black_wires` to allow to keep empty classical bits
```

Comment:
Small typo `remove_blank_wires`

Reply:
Thank you! done in [937af3b](https://github.com/CQCL/tket/pull/1435/commits/937af3ba1e202c212ba36460eea1b1bc4abfd506)